pkg/daemon/ceph/client/filesystem.go

```diff
@@ -168,6 +169,10 @@ func AddDataPoolToFilesystem(context *clusterd.Context, clusterInfo *ClusterInfo
 	args := []string{"fs", "add_data_pool", name, poolName}
 	_, err := NewCephCommand(context, clusterInfo, args).Run()
 	if err != nil {
+		// TODO: remove this once https://github.com/rook/rook/issues/12673 is fixed.
+		if code, ok := exec.ExitStatus(err); ok && code == int(syscall.EINVAL) {
```

Comment:
Let's only ignore this code for Reef: `if clusterInfo.CephVersion.IsAtLeastReef()`